.jsdoc.js

```diff
@@ -31,18 +34,19 @@ module.exports = {
   source: {
     excludePattern: '(^|\\/|\\\\)[._]',
     include: [
-      'build/src'
+      'build/src',
+      'protos'
     ],
     includePattern: '\\.js$'
   },
   templates: {
-    copyright: 'Copyright 2019 Google, LLC.',
+    copyright: 'Copyright 2020 Google LLC',
     includeDate: false,
     sourceFiles: false,
-    systemName: '@google-cloud/spanner',
+    systemName: 'spanner',
     theme: 'lumen',
```

Comment:
Just curious, we decided to rename it here to `spanner`?

Reply:
Hmmmmm

Reply:
Nice catch! It's the missing `package-name`. Fixed!

Reply:
Thank you! congrats for the last API conversion! :)